css/ampshared.css

```diff
@@ -322,9 +322,12 @@ amp-img:not(.i-amphtml-element)[i-amphtml-ssr] > img.i-amphtml-fill-content
 .i-amphtml-element > [placeholder],
 [layout]:not(.i-amphtml-element) > [placeholder],
 [width][height][sizes]:not([layout]):not(.i-amphtml-element) > [placeholder],
-[width][height][heights]:not([layout]):not(.i-amphtml-element) > [placeholder]
-{
+[width][height][heights]:not([layout]):not(.i-amphtml-element) > [placeholder] {
   display: block;
+  /* line-height:normal overrides line-height:0 from the "hide all children of
+     non-container text node children" rule. This avoids a minor content jump during
+     element build since placeholders should be fully visible. */
+  line-height: normal;
```

Comment:
Would an alternative approach to add `:not([placeholder])` to the original selectors which apply `line-height: 0` be preferred instead?:
https://github.com/ampproject/amphtml/blob/de69bf763514d370ea383e5563beee8f4b362cfe/css/ampshared.css#L297-L304

Reply:
I'm not sure that would work, since the original selector was targetting the amp-element itself.